homeassistant/components/schlage/quality_scale.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
```

Comment:
We are not testing the unique id in the test_form test. (Also, the name and the docstring is out of date)

Reply:
Also `test_form_requires_unique_id`, we just added it but the name is strange now that I look at it